packages/stack/contracts-manager/src/index.js

```diff
@@ -552,6 +551,28 @@ export default class ContractsManager {
     });
   }
 
+  _isParentParentContractDependencyCorrect(className, parentContract, typeOfInheritance, dictionary) {
```

Comment:
I like how no one noticed that I put the word Parent twice by accident in the function name haha.